Makefile

```diff
@@ -247,6 +248,17 @@ test-ci-integration:
 
 define SUBDIR_RULES
 
+# We override the rules for `*-gen-kube` to just generate the kube manifest
+# bundle.
+ifeq ($(SUBDIR), kube)
+
+# Builds the single kube bundle from individual manifest files.
+all-gen-kube:
+	@echo "--- Generating kube bundle"
+	@./kube/scripts/build_bundle.sh
```

Comment:
hm is there a way to verify if the generated bundle file is valid?

Reply:
Hmm, do you mean like valid Kube objects or just not garbage YAML? I'm not sure of a way to do k8s validation without a remote API server and `kubectl` install.

Reply:
ideally kube level, for e.g. dunno how good it is but something like https://github.com/garethr/kubeval maybe might be nice?

Feel free to do as a followup, don't want to derail this PR

Reply:
Ah nice, I like that idea. Will follow up.